Converters/ArcGIS/Speckle.Converters.ArcGIS3/Utils/ArcGISFieldUtils.cs

```diff
@@ -80,14 +80,14 @@ public RowBuffer AssignFieldValuesToRow(
     return rowBuffer;
   }
 
-  public List<FieldDescription> GetFieldsFromSpeckleLayer(GisLayer target)
+  public List<FieldDescription> GetFieldsFromSpeckleLayer(Collection target)
   {
-    if (target["attributes"] is Base attributes)
+    if (target["fields"] is Dictionary<string, string> attributes)
```

Comment:
do we attach properties to collections in arcgis?

Reply:
yes, some applications like arcgis and navis have meaningful properties on collections (not jsut objects) - we don't have a convention for handling this, could be some `DataCollection: IDataCollection:IProperties` class in the future